### DISCUSSION TITLE
How to trigger the validation on an input when the value of a related input changes?

### DISCUSSION THREAD

**Discussion by costinEEST** [react-hook-form/react-hook-form#5887] [Show and tell]
How to trigger the validation on the `confirmEmail` field, every time when the value of the `email` field changes?
Here is a working solution - https://codesandbox.io/s/trigger-field-validation-when-related-field-value-changes-reacthookfom-s208q 
Somehow I want to extract all the logic for the `<input />` in a reusable component (something which exposes the `ref`, `onBlur` and `onChange` plus some custom `props`).
Tried this - https://codesandbox.io/s/validate-field-when-other-one-changes-react-hook-form-e5vg4 without success.

**Comment by bluebill1049** [comment]
I think it may be worth sending a feature request with this, as it's been discussed multiple times. the relevant one here as well: https://github.com/react-hook-form/react-hook-form/discussions/5597

I think the idea is passing down `dep` as an array of pathName (string), then onChange can just fire the those dep's validation
